<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Courtney Cavin</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dismember</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTDataView" name="e_dataview" native="true">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="T_main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="t_disassembly">
       <attribute name="title">
        <string>Disassembly</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTCodeView" name="e_assembly">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTSearchWidget" name="e_searchwidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="t_tree">
       <attribute name="title">
        <string>Tree</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="t_graph">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="M_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="a_manual"/>
    <addaction name="a_about"/>
   </widget>
   <widget class="QMenu" name="m_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="a_undo"/>
    <addaction name="a_find"/>
    <addaction name="a_findnext"/>
    <addaction name="a_findprev"/>
   </widget>
   <widget class="QMenu" name="m_tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="a_w_symbols"/>
    <addaction name="a_w_tools"/>
    <addaction name="a_w_datatypes"/>
    <addaction name="a_w_terminal"/>
   </widget>
   <widget class="QMenu" name="m_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="a_open"/>
    <addaction name="a_save"/>
    <addaction name="a_saveas"/>
    <addaction name="a_quit"/>
   </widget>
   <addaction name="m_file"/>
   <addaction name="m_edit"/>
   <addaction name="m_tools"/>
   <addaction name="m_help"/>
  </widget>
  <widget class="QStatusBar" name="w_statusbar">
   <widget class="QTRunQueueMonitor" name="e_monitor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="w_symbols">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>106</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Symbols</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QTSymbolView" name="e_symbolview">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="w_tools">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QListView" name="e_toolsview">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="w_datatypes">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Data Types</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QTDataTypeView" name="e_datatypeview">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="w_terminal">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Python Terminal</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QTPythonTerminalView" name="e_terminalview">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <action name="a_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_open</normaloff>:/icons/action_open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="a_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_save</normaloff>:/icons/action_save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="a_saveas">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_saveas</normaloff>:/icons/action_saveas</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="a_quit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_quit</normaloff>:/icons/action_quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="a_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="a_w_symbols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Symbols</string>
   </property>
   <property name="iconText">
    <string>Symbols</string>
   </property>
   <property name="toolTip">
    <string>Symbols</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="a_w_tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tools</string>
   </property>
  </action>
  <action name="a_w_datatypes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Data Types</string>
   </property>
  </action>
  <action name="a_manual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_help</normaloff>:/icons/action_help</iconset>
   </property>
   <property name="text">
    <string>&amp;Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="a_about">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_about</normaloff>:/icons/action_about</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="a_w_terminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>T&amp;erminal</string>
   </property>
  </action>
  <action name="a_find">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/action_find</normaloff>:/icons/action_find</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="a_findnext">
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="a_findprev">
   <property name="text">
    <string>Find &amp;Previous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTCodeView</class>
   <extends>QTableView</extends>
   <header>codeview.h</header>
  </customwidget>
  <customwidget>
   <class>QTDataView</class>
   <extends>QWidget</extends>
   <header>dataview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTSymbolView</class>
   <extends>QListView</extends>
   <header>symbolview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTRunQueueMonitor</class>
   <extends>QLabel</extends>
   <header>runqueuemonitor.h</header>
  </customwidget>
  <customwidget>
   <class>QTDataTypeView</class>
   <extends>QListWidget</extends>
   <header>datatypeview.h</header>
  </customwidget>
  <customwidget>
   <class>QTPythonTerminalView</class>
   <extends>QPlainTextEdit</extends>
   <header>pythonterminalview.h</header>
  </customwidget>
  <customwidget>
   <class>QTSearchWidget</class>
   <extends>QWidget</extends>
   <header>searchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>T_main</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>a_w_symbols</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_symbols</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a_w_datatypes</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_datatypes</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a_w_tools</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_tools</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_symbols</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>a_w_symbols</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_tools</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>a_w_tools</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>813</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_datatypes</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>a_w_datatypes</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>813</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a_w_terminal</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_terminal</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_terminal</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>a_w_terminal</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
